src/lib/hooks/useMinimalShellTransform.ts

```diff
@@ -27,8 +28,13 @@ export function useMinimalShellHeaderTransform() {
 export function useMinimalShellFooterTransform() {
   const mode = useMinimalShellMode()
   const {footerHeight} = useShellLayout()
+  const gate = useGate()
+  const isFixedBottomBar = gate('fixed_bottom_bar')
```

Comment:
Have to check here because can't do on another thread. It'll have behavior immediately on scroll so seems fine.